interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/nodecompilation/ValueEditorValidator.scala

```diff
@@ -12,21 +12,20 @@ import pl.touk.nussknacker.engine.api.definition.{
   ParameterEditor
 }
 import pl.touk.nussknacker.engine.api.editor.DualEditorMode
-import pl.touk.nussknacker.engine.graph.node.FragmentInputDefinition.{FragmentClazzRef, ValueInputWithFixedValues}
+import pl.touk.nussknacker.engine.graph.node.FragmentInputDefinition.{
```

Comment:
1. I just realize that you still keep these classes inside `FragmentInputDefinition`. We need to move them out from there. They become not only Fragment specific. 
2. We have a dedicated module (`common-api`) which was intended to be used in such situation when we have some part of api shared between components and scenario graph. Let's move `FixedExpressionValue` class there and reuse it instead of duplicating